src/App.js

```diff
@@ -1,23 +1,45 @@
-import logo from './logo.svg';
 import './App.css';
+import { useState } from 'react';
+import OrderImages from './OrderImages';
+import OrderNameInput from './OrderNameInput';
+import OrderSelector from './OrderSelector';
+import OrderInstructionForm from './OrderInstructionForm';
+import OrderInstructions from './OrderInstructions';
 
 function App() {
+  const [orderName, setOrderName] = useState('');
+  const [burgerId, setBurgerId] = useState('1');
+  const [drinkId, setDrinkId] = useState('1');
+  const [sideId, setSideId] = useState('1');
+  const [orderInstructions, setOrderInstructions] = useState([]);
+  const [instructionInput, setInstructionInput] = useState(['']);
```

Comment:
We don't need empty strings inside of the array on line 15, just the empty strings since this is the input we're tracking and not all of the instructions like on 14